src/DataAPI.jl

```diff
@@ -161,8 +161,13 @@ end
 
 """
     Cols(cols...)
+    Cols(f::Function)
 
 Select the union of the selections in `cols`. If `cols == ()`, select no columns.
+
+If the only positional argument is a `Function` `f` then select the columns whose
+names passed to the `f` predicate as strings return `true`.
+As a special case if `:` is passed (the `Colon` function) select all columns.
```

Comment:
Do we really need to mention `:`? It sounds quite obvious that if `:` is allowed as a selector to get all columns, passing it selects all columns.
```suggestion
```

Reply:
Technically `:` is a `Function` so this is a special case. Do you still think it should be removed?

Reply:
Ah, got it! I wouldn't mention it though, as nobody would expect `:` to be treated as a predicate on strings (anyway that would error).

Reply:
OK - removing.